Examples/BugReport/Package.swift

```diff
@@ -0,0 +1,24 @@
+// swift-tools-version:4.2
```

Comment:
we should support the same versions the driver works with, 4.0+. I believe this won't work on lower versions. I think you can provide two separate `Package.swift` files for different swift-tools-versions, I've seen people do it in repos before. probably info in the SPM docs

Reply:
Does it make sense to include `Package.resolved` in the example?

Reply:
Good catch, I missed this because I've been running Swift 4.2.

Reply:
I've just bumped it down to 4.0.